isaric/parsers/ccpuk_observations.toml

```diff
@@ -0,0 +1,465 @@
+[adtl.defs.admissionDateHierarchy]
+combinedType = "firstNonNull"
+        fields = [
+            { field = "hostdat", description = "Admission date at this facility" },
+            { field = "cestdat", description = "Onset date of first/earliest symptom" },
+            { field = "dsstdat", description = "Date of enrolment" },
+            { field = "daily_dsstdat", description = "Date of assessment" }
+        ]
+
+[adtl.defs.followupDateHierarchy]
+combinedType = "firstNonNull"
+fields = [
+{ field = "flw_survey_date", description = "About you and your COVID-19 illness (if you're completing this survey on behalf of a child or adult that you care for, all the questions relate to their health and wellbeing)" },
+{ field = "dsstdtc", description = "Outcome date", if = { dsterm = 4 } },
+{ field = "dsstdtc_v2", description = "Outcome date", if = { dsterm_v2 = 4 } }
+]
+
+[[observation]]
+name = "haemorrhage"
+date = { ref = "observationDateHierarchy" }
+phase = "admission"
+is_present = { field = "cough_ceoccur_v2", ref = "Y/N/NK" }
```

Comment:
something else for field?

Reply:
Yes. I think I have changed this .. or will change shortly.